app_dart/lib/src/model/firestore/ci_staging.dart

```diff
@@ -293,37 +308,52 @@ enum CiStage implements Comparable<CiStage> {
   String toString() => name;
 }
 
+enum StagingConclusionResult {
+  /// Check run update recorded successfully in the respective CI stage.
+  ok,
```

Comment:
Nit: maybe add:

/// It is OK to evaluate returned results for stage completeness